nimare/stats.py

```diff
@@ -148,10 +148,8 @@ def null_to_p(test_value, null_array, tail="two"):
         raise ValueError('Argument "tail" must be one of ["two", "upper", ' '"lower"]')
 
     smallest_value = np.maximum(np.finfo(float).eps, 1.0 / len(null_array))
-    if p_value == 0:
-        p_value = smallest_value
-    elif p_value == 1:
-        p_value = 1.0 - smallest_value
+
+    p_value = max(smallest_value, min(p_value, 1.0 - smallest_value))
```

Comment:
This is a very dense line. I don't want you to write less efficient code to improve readability, so would you mind adding a comment above it?

Reply:
done!